layers/meta-resin-raspberrypi/conf/samples/local.conf.sample

```diff
@@ -16,6 +16,9 @@ GPU_MEM = "16"
 # RPI BSP uses uncompressed kernel images by default
 KERNEL_IMAGETYPE="zImage"
 
+# RPI Use u-boot. Set this to 1 to use u-boot
```

Comment:
I'm not sure why you add it here and afterwards change it to 1 in the next commit. I would just set it to 1 in the next commit.

Reply:
That was a sneaky way of letting jenkins build test without u-boot and then with u-boot